common/lib/xmodule/xmodule/x_module.py

```diff
@@ -603,8 +603,9 @@ def editable_metadata_fields(self):
         fields = getattr(self, 'unmixed_class', self.__class__).fields
 
         for field in fields.values():
-
-            if field.scope != Scope.settings or field in self.non_editable_metadata_fields:
+            if field in self.non_editable_metadata_fields:
+                continue
+            if field.scope not in (Scope.settings, Scope.content):
```

Comment:
I know there's been a lot of talk about including Scope.content fields in the editor, so I think this is a good idea. It makes me a little nervous in that we will have to check every editor for every xblock/xmodule to ensure that an unexpected field hasn't slipped in. Do we have unit and/or Bok Choy tests that verify the fields shown?